spec/linter-reek-spec.js

```diff
@@ -20,7 +20,7 @@ describe('The reek provider for Linter', () => {
 
   it('checks a file with issues and reports the correct message', async () => {
     const excerpt = 'IrresponsibleModule: Dirty has no descriptive comment';
-    const url = 'https://github.com/troessner/reek/blob/master/docs/Irresponsible-Module.md';
+    const url = 'https://github.com/troessner/reek/blob/v5.3.1/docs/Irresponsible-Module.md';
```

Comment:
This should probably be a regex to handle changing versions without needing to update the specs.